### PR TITLE
feat: add perpetual banner

### PR DESCRIPTION
**Pull request by chef-eric**
---

## PR-Codex overview
The focus of this PR is to update banner components and translations, add a new PerpetualSeasonalBanner, and refactor useMultipleBannerConfig.

### Detailed summary
- Updated `BannerTitle` component to handle additional variant types
- Added `Trade and Win` banner with translations and links
- Introduced `PerpetualSeason` and `PerpetualSeasonalBanner` components
- Refactored `useMultipleBannerConfig` to include `PerpetualSeasonalBanner` and remove `PerpetualBanner` usage

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/views/Home/components/Banners/images/ApolloXCampaignLogo.png
Comment (chefjackson) [inline apps/web/src/views/Home/components/Banners/images/ApolloXCampaignLogo.png:1]: let's move assets to the assets repo

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ export const useMultipleBannerConfig = () => {
       },
       {
         shouldRender: true,
-        banner: <PerpetualBanner />,
+        banner: <PerpetualSeasonalBanner />,
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx:88]: Move this under `ListaIFOBanner`

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx [hunk 2]
```diff
@@ -47,7 +46,10 @@ export const useMultipleBannerConfig = () => {
         shouldRender: isRenderUserBanner.shouldRender && !isRenderUserBanner.isEarningsBusdZero,
         banner: <UserBanner />,
       },
-      { shouldRender: shouldRenderIfoBanner, banner: <ListaIFOBanner /> },
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx:50]: Don't delete the `ListaIFOBanner` just list under it.

File: apps/web/src/components/PhishingWarningBanner/PerpetualSeason.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  return (
+    <Flex mr={['6px']} alignItems="center" flexWrap="wrap">
+      {content}
+      <FlexGap>
+        <Link
+          color="primary"
+          fontSize={['12px', '12px', '14px']}
+          style={{ display: 'flex', alignItems: 'center', whiteSpace: 'nowrap' }}
+          data-dd-action-name="stripe-InterfacefeeRefund"
+          href="https://perp.pancakeswap.finance/en/futures/v2/BTCUSD?utm_source=infostripe&utm_medium=website&utm_campaign=PerpARBIncentives&utm_id=ARBincentives"
```
[the remark this answers is not in the document]
Comment (chef-eric) [reply apps/web/src/components/PhishingWarningBanner/PerpetualSeason.tsx]: done

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx (hunk 1 above)
Comment (chef-eric) [reply apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx]: done

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx (hunk 2 above)
Comment (chef-eric) [reply apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx:50]: @chefjackson say Lista can delete now

File: apps/web/src/views/Home/components/Banners/images/ApolloXCampaignLogo.png
Comment (chef-eric) [reply apps/web/src/views/Home/components/Banners/images/ApolloXCampaignLogo.png:1]: https://github.com/pancakeswap/assets/pull/76/files

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx (hunk 2 above)
Comment (ChefMomota) [reply apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx:50]: Can u comment out the code `shouldRenderIfoBanner` in line  `17`,`41`,`101`

Comment (chef-eric) [reply apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx:50]: done

File: apps/web/src/views/Home/components/Banners/PerpetualSeasonalBanner.tsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+import styled from 'styled-components'
+import { useTheme } from '@pancakeswap/hooks'
+import { getPerpetualUrl } from 'utils/getPerpetualUrl'
+import { useMemo } from 'react'
+
+const bgMobile = `${ASSET_CDN}/web/banners/perpetual-season-banner/bunny-bg-mobile.png`
+const bgDesktop = `${ASSET_CDN}/web/banners/perpetual-season-banner/bunny-bg.png`
+const floatingAsset = `${ASSET_CDN}/web/banners/perpetual-season-banner/floating.png`
+const arbLogo = `${ASSET_CDN}/web/banners/perpetual-season-banner/arbSimplifiedLogo.png`
+const apxLogo = `${ASSET_CDN}/web/banners/perpetual-season-banner/apolloXCampaignLogo.png`
```
Comment (chefjackson) [inline apps/web/src/views/Home/components/Banners/PerpetualSeasonalBanner.tsx:28]:
```suggestion
const apxLogo = `${ASSET_CDN}/web/banners/perpetual-season-banner/ApolloXCampaignLogo.png`
```

Comment (chefjackson) [reply apps/web/src/views/Home/components/Banners/PerpetualSeasonalBanner.tsx]: and let's unify the naming convention next time. Better keep the same pattern. Prefer kebab case for asset filenames

**Review by chefjackson — CHANGES_REQUESTED** [review]
The logo image is broken. Please check the asset path

**Review by chefjackson — APPROVED** [review]

**Pull request merged**